<launch>
  <node launch-prefix="gdb -ex run --args" pkg="pointcloud_filter" type="wall_detection" name="wall_detection" output="screen" ns="red"/>
</launch>